templates/WG-README.md

```diff
@@ -0,0 +1,83 @@
+# CHAOSS \<_wg-name_\> Working Group
+
+Badges (If Any)
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Participate](#participate)
+- [Contributing](#contributing)
+- [Metrics](#metrics)
+- [Contributors](#contributors)
+- [License](#license)
+
+## Introduction
+
+### Goals 
+
+A brief description about the goals of the working group.
+<br/>
+
+### Purpose
+
+Answers to why the working group was formed.
+<br/>
+
+### Who should join this working group?
+
+Interests and requirements of the working group.
+<br/>
+
+### Background
+
+- History of the working group
+- Name changes in past (if any)
+- Any other relevant past information
+  
+## Participate
+
+### How to Join Us?
+
+A section guiding the user on how to join the working group.
+
+- Link to the Agenda/Meeting-Minutes document
+- Link to the [Participate](https://chaoss.community/participate/) page on the website
+
+Link to [Code of Conduct](https://github.com/chaoss/governance/blob/master/code-of-conduct.md)
+
+## Contributing
+
+Link to `CONTRIBUTING.md` of the concerned working group.
+
+## Metrics
+
+### Focus-Areas and Goals
+Mainly a table of the focus areas and their goals linked to subfolders in the `focus-areas` folder. 
+<br/>
+
+### Released Metrics
+A section guiding the user to the released metrics.
+
+- Link to the [metrics](https://chaoss.community/metrics/) page on the website
+
+Link to the [translations](https://github.com/chaoss/translations) repository.
+
+## Contributors
+
+### Chairs
```

Comment:
```suggestion
### WG Maintainers
```

Reply:
As discussed in the meeting, we want to keep Chairs separate from the maintainers.